player/types.go

```diff
@@ -9,3 +11,46 @@ type PartInfo struct {
 	Player string
 	Part   int64
 }
+
+type PlayerRound struct {
+	ID        int64
+	Status    int64
```

Comment:
We need the RoundId (1-10) here somewhere

Reply:
Added